src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateGoalTest.kt

```diff
@@ -32,7 +32,7 @@ import uk.gov.justice.digital.hmpps.educationandworkplanapi.withBody
 class UpdateGoalTest : IntegrationTestBase() {
 
   companion object {
-    private const val URI_TEMPLATE = "/action-plans/{prisonNumber}/goals/{goalReference}"
+    const val URI_TEMPLATE = "/action-plans/{prisonNumber}/goals/{goalReference}"
```

Comment:
Not that I massively care 🤣 , but is there a change somewhere that I've missed that uses this? Or is it a change as a result of some earlier work?

Reply:
yes this can be privatised.